common/app/conf/switches/FeatureSwitches.scala

```diff
@@ -316,6 +316,16 @@ trait FeatureSwitches {
     exposeClientSide = true
   )
 
+  val DiscussionOrderByOldest = Switch(
+    SwitchGroup.Feature,
+    "discussion-order-by-oldest",
+    "If on, comments default sort ordering will be oldest first. If off, then the default is newest first",
+    owners = Seq(Owner.withGithub("nicl")),
+    safeState = Off,
+    sellByDate = never,
+    exposeClientSide = true
+  )
+
```

Comment:
I would still make it an [A/B test switch](https://github.com/guardian/frontend/blob/master/common/app/conf/switches/ABTestSwitches.scala) because it's an A test of sorts.

The only difference is that it makes it clear that this isn't a feature, just something you're testing (which I think is a valuable distinguisher).